app/backend/db/models/date_datas.py

```diff
@@ -22,9 +20,30 @@ class DateDetails(Base):
 class DateData(Base):
     __tablename__ = 'date_datas'
     id = Column(Integer, primary_key=True, unique=True)
-    date = Column(Date, ForeignKey('date_details.date', ondelete='CASCADE'), default=date.today(), index=True)
+    date = Column(Date, ForeignKey('date_details.date', ondelete='CASCADE'), index=True)
     date_details = relationship('DateDetails', backref='datas')
     user_id = Column(Integer, ForeignKey('users.id', ondelete='CASCADE'), index=True)
-    user = relationship('User', backref='history')
+    user = relationship('User', backref='statuses')
     state = Column(Enum('here', 'not_here', name='answer_state'))
-    reason = Column(Integer, ForeignKey('reasons.id'), index=True)
+    reason_id = Column(Integer, ForeignKey('reasons.id'), index=True)
+    reason = relationship('Reason', backref='date_datas')
```

Comment:
Billable in case of Repetative/ just get date's type (if you need to fill it futurely or something)

Reply:
what is billable?